src/autoplex/misc/qe/run.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+import logging
+import os
+import re
+import subprocess
+
+from ase.io import read
+from ase.units import GPa
+from jobflow import job
+from pymatgen.io.ase import AseAtomsAdaptor
+
+from .schema import InputDoc, OutputDoc, TaskDoc
+
+logger = logging.getLogger(__name__)
+
+
+_ENERGY_RE = re.compile(r"!\s+total energy\s+=\s+([-\d\.Ee+]+)\s+Ry")
+
+
+def _parse_total_energy_ev(pwo_path: str) -> float | None:
+    """
+    Extract and return total energy (eV) if found in QE output (.pwo)
+
+    Parameters
+    ----------
+    pwo_path : str
+        Path to QE output file (.pwo)
+
+    Returns
+    -------
+    float | None
+        Total energy in eV if found, else None
+    """
+    if not os.path.exists(pwo_path):
+        return None
+
+    try:
+        with open(pwo_path, errors="ignore") as fh:
+            for line in fh:
+                m = _ENERGY_RE.search(line)
+                if m:
+                    # convert energy in eV
+                    ry = float(m.group(1))
+                    return ry * 13.605693009
+    except Exception:
+        return None
+    return None
+
+
+@job
+def run_qe_static(input: InputDoc, command: str) -> TaskDoc:
```

Comment:
Is this specifically just for static? Seems like a generic QE call. Maybe rename this?